neuromation/api/config.py

```diff
@@ -455,7 +458,20 @@ def _validate_user_config(
     # additional API for describing new supported config sections, keys and values.
     # Right now this functionality is skipped for the sake of simplicity.
     _check_sections(config, {"alias", "job", "storage"}, filename)
-    _check_section(config, "job", {"ps-format": str}, filename)
+    _check_section(
+        config,
+        "job",
+        {
+            "ps-format": str,
+            "default-life-span": {
```

Comment:
Please use `"life-span": str` here, where `str` is exactly the same string as passed to `neuro run --life-span TIMEDELTA`.

Reply:
ok